src/SimpleSAML/XHTML/Template.php

```diff
@@ -217,7 +218,7 @@ public function asset(string $asset, ?string $module = null, bool $tag = true):
             'assets.secretsalt.default',
         );
         $mac = hash_hmac('sha256', $tag, $salt, true);
```

Comment:
If we set the fourth parameter to `false`, we are given a url-safe hash and we don't need the added `base64_url_encode` utility method.